_posts/2021-12-31-medical-diagnosis.md

```diff
@@ -0,0 +1,87 @@
+# Validated Pattern: Medical Diagnosis
+---
+layout: post <br>
+date: 2021-12-31 <br>
+title: Medical Diagnosis <br>
+published: true <br> 
+tags:
+- medical-diagnosis
+- xray
+- validated-pattern
+---
+Our team recently completed the development of a validated pattern that showcases the capabilities we have at our fingertips when we combine OpenShift and other cutting edge Red Hat technologies to deliver a solution. 
+
+We've taken an application defined imperatively in an Ansible playbook and converted it into GitOps style declarative kubernetes resources. Using the validated pattern framework we are able to deploy, manage and integrate with multiple cutting edge Red Hat technologies, and provide a capability that the initial deployment strategy didn't have available to it: a lifecycle. Everything you need to take this pattern for a spin is in [git](https://github.com/hybrid-cloud-patterns/medical-diagnosis).
+
+#### Pattern Workflow
+The purpose of this pattern is to show how medical facilities can take full advantage of trained AI/ML models to identify anomalies in the body like pneumonia. From the medical personnel point of view it works with medical imagery equipment submitting an X-ray image into the application to start the workflow. 
+
+The image is uploaded to an S3-compatible object storage. This upload triggers an event from the storage, “a new image has been uploaded”,  that is sent into a Kafka topic. This topic is consumed by a KNative Eventing listener that triggers the launch of a KNative Serving instance. This instance is a containerimage  with the AI/ML model and the needed processing functions. Based on the information from the event it received, the container retrieves the image from the object store, pre-processes it, makes a prediction on the risk of pneumonia using the AI/ML model, and saves the result. A notification of those results is sent to the medical staff as well.
+
+![](https://hybrid-cloud-patterns.io/images/medical-edge/physical-dataflow.png?raw=True)
+
+For a recorded demo deploying the pattern and seeing the dashboards available to the user, check out our [docs page](https://hybrid-cloud-patterns.io/medical-diagnosis/)!
+
+#### Pattern Deployment
+---
+Before getting started, you need to install the following packages and collections.
+**PREREQUISITES** <br>
+
+| Packages |
+|:--------:|
+| git |
+| make |
+| ansible |
```

Comment:
helm? oc?

Reply:
Actually, let's not even put any of this here.  Lets just link to the install instructions

Reply:
@beekhof - i actually spun up a fresh vm so I could walk all the way through ( to catch assumptions ) and I hit this with oc and helm ... will link to install instructions